Orange/preprocess/score.py

```diff
@@ -30,12 +31,37 @@ class Scorer(_RefuseDataInConstructor, Reprable):
         RemoveNaNClasses()
     ]
 
+    @property
```

Comment:
Sadly, Python doesn't provide `@classproperty` (and, no, you can't combine `@classmethod` and `@property`). Implementing it doesn't make sense for just a few (overpedantic) uses. :)